core/src/test/scala/com/raphtory/enrontest/RaphtoryENRONTest.scala

```diff
@@ -1,41 +1,36 @@
 package com.raphtory.enrontest
 
 import com.raphtory.BaseRaphtoryAlgoTest
-import com.raphtory.GraphState
-import com.raphtory.Raphtory
 import com.raphtory.algorithms.generic.ConnectedComponents
 import com.raphtory.api.input.GraphBuilder
 import com.raphtory.api.input.Spout
-import com.raphtory.internals.components.spout.SpoutExecutor
 import com.raphtory.sinks.FileSink
 import com.raphtory.spouts.FileSpout
-import org.apache.pulsar.client.api.Schema
-import org.scalatest.DoNotDiscover
 
+import java.net.URL
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.Duration
+import scala.concurrent.duration.FiniteDuration
 import scala.language.postfixOps
-import sys.process._
-import java.io.File
 
-@DoNotDiscover
 class RaphtoryENRONTest extends BaseRaphtoryAlgoTest[String] {
 
-  test("Graph State Test") {
-    val sink: FileSink = FileSink(outputDirectory)
-
-    graph
-      .walk(10000)
-      .past()
-      .execute(GraphState())
-      .writeTo(sink)
-      .waitForJob()
-
-    assert(true)
-  }
+//  withGraph.test("Graph State Test") { graph =>
```

Comment:
Why was this test turned off? Is something up with walk? (I know this was DoNotDiscover before, but is this something to fix in the future?)

Reply:
this thing keeps running foreva!

Reply:
Ok that is probably a case of us having to dig into if walk is working correctly then